Update RapidUpload, add rapidUploadV2

reveever opened the pull request:
实现了新秒传接口

reveever committed:
Update RapidUpload, add rapidUploadV2

mengzonefire commented:
create步骤中的如下参数其实是无效的:
		"a":          "commit",
		"sequence":   "1",
		"autoinit":   "1"

*源于最初开发时误参考了旧版文档, 新版文档在 [这里](https://pan.baidu.com/union/document/basic#创建文件)

reveever committed:
Update req bdoy for PrepareRapidUploadV2
https://pan.baidu.com/union/document/basic#预上传
https://pan.baidu.com/union/document/basic#创建文件

reveever commented:
已修改
原来这个就是分片上传的接口，怪不得我看有的结构都是现成的

qjfoidnh commented:
试了两个之前有问题的秒传，都成功了，感谢两位的工作

reveever commented:
似乎有 bug，上传一个全新文件时，默认尝试秒传，结果秒传没有返回错误，之后看看什么原因

mengzonefire commented:
> 似乎有 bug，上传一个全新文件时，默认尝试秒传，结果秒传没有返回错误，之后看看什么原因

我在issue内详细注明了: https://github.com/qjfoidnh/BaiduPCS-Go/issues/106
precreate是不会返回报错的, 需要根据返回的block_list是否为空判断秒传有效, 然后主动返回错误

reveever commented:
嗯 单纯的代码错误，判断逻辑没问题，已经修复

mengzonefire commented:
今天测试了下发现:

新接口可以简化, 直接create即可, 不需要precreate, uploadid参数可以省去无任何影响(create返回errno: 2为不识别秒传)

顺便, 新接口是支持20G以上文件的, 可以添加下支持
